scripts/deploy_kakarot.py

```diff
@@ -2,7 +2,12 @@
 import logging
 from asyncio import run
 
-from scripts.constants import COMPILED_CONTRACTS, ETH_TOKEN_ADDRESS, EVM_ADDRESS
+from scripts.constants import (
+    COMPILED_CONTRACTS,
+    ETH_TOKEN_ADDRESS,
+    DEPLOY_FEE,
```

Comment:
run `make format` I guess that this doesn't pass

Reply:
done!